src/main/scala/group/scala/karazin/mongo4s/model.scala

```diff
@@ -66,12 +66,12 @@ object model:
 
   final case class WriteConcern(w: Int, j: Boolean, wtimeout: Long) derives Codec.AsObject
 
-  final case class Insert[V, Comment](insert: String,
+  final case class Insert[V](insert: String,
                              documents: List[V],
                              ordered: Option[Boolean] = None,
                              writeConcern: Option[WriteConcern] = None,
                              bypassDocumentValidation: Option[Boolean] = None,
-                             comment: Option[Comment] = None) derives Codec.AsObject
+                             comment: Option[JsonObject] = None) derives Codec.AsObject
```

Comment:
But wouldn't just Json be appropriate?

Reply:
You're right. I again confused these types.